tests/test_generators/test_pythongen.py

```diff
@@ -68,6 +68,10 @@ def test_multiline_stuff(input_path):
     )
 
 
+def test_ifabsent_stuff(input_path):
```

Comment:
sorry should have added this to the previous review - what does this do?

Reply:
It simply tries to generate the Python code corresponding to the schema and it fails without the patch. It is a sort of "poor-man" regression test.